packages/logstash/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.4.1"
+  changes:
+    - description: Add missing host.ip field mapping
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/1
```

I think here you want to link the current PR right?
```suggestion
      link: https://github.com/elastic/integrations/pull/8526
```

---

Thanks for catching that!